graph_altimeter/scan/postprocess.py

```diff
@@ -37,6 +37,7 @@ def _create_vertex(v_id, scan_id, account_id):
         "scan_id": scan_id,
         "account_id": account_id,
         "arn": str(v_id),
+        "missing": "yes"
```

Comment:
```suggestion
        "missing": True
```

What about using a boolean?

Reply:
We could use it but by now it doesn't actually matter, in the sense that [here](https://github.com/adevinta/graph-altimeter/blob/3afc5ceeab35f7b6e8db60b5dde9b2daecb29111/graph_altimeter/scan/neptune_client.py#L79) we are converting all the property values to string. In any case I'm changing because it's maybe cleaner.